brands/amenity/fast_food.json

```diff
@@ -211,7 +211,10 @@
       "brand:wikidata": "Q177054",
       "brand:wikipedia": "en:Burger King",
       "cuisine": "burger",
-      "name": "Burger King"
+      "name": "Burger King",
+      "operator": "Restaurant Brands International",
```

Comment:
wait, aren't most of these operated by franchisees?
`operator` isn't really for "parent company"

Reply:
Probably. I'll remove them. I'm probably a little loser in my usage of the operator tag then I could be, but there's no owner:wikipedia tag and Wikipedia linking is the main reason I use it :(